src/EnergyPlus/HVACCooledBeam.cc

```diff
@@ -946,32 +918,31 @@ namespace HVACCooledBeam {
         int ControlNode;              // the water inlet node
         int InAirNode;                // the air inlet node
         bool UnitOn;                  // TRUE if unit is on
-        int SolFlag;
```

Comment:
[src/EnergyPlus/HVACCooledBeam.cc:949]:(style),[variableScope],The scope of the variable 'SolFlag' can be reduced.